### PR TITLE
Character creation guidelines

### PR DESCRIPTION
**Pull request by djigneo**
Updated guidelines to remove language around Session 0 and DMs.

### CODE REVIEW

**Comment by djigneo** [comment]
Sorry for the file conflicts. My Git skills apparently aren't that strong.

I'm not sure what to say about the Issue template. Perhaps it would be helpful to separate the mandatory checklist steps from the "helpful to think about" steps, with the mandatory steps going into the Issue template?

The way I see it, the "step-by-step characters" such as it was are guidelines for helping a player think through the character, but they are really _optional_ steps to making a character that's playable in this game. It's a supplement, not a requirement, so perhaps the checklist which appears in the Issue template are all the absolutely mandatory parts.

To that end, it perhaps makes sense to split out the character creation parts that are more like "rules" into a separate document as well to make an explicit distinction between the guidelines for the game and supplemental information to help with character creation.

**Comment by zmoazeni** [comment]
Yeah I'm totally with you. I like the idea of us splitting that way:

1. The stuff in the issue template mentions the things that are mandatory while hinting at the things encouraged. It also links to other areas in the repository for more information.
2. I do really like the vast majority of the content you posted here. I'll go through it a little more thoroughly later, but I feel they compliment each other well. The README will link to this, and perhaps so will different sections of the issue template to help guide the person.

I really just want to avoid someone having to read _everything_ just to join a game. If they can start by creating an issue template and it communicates everything they need to do, that sounds like it'll be swell.

For the record, I want to do the exact same thing with the session template. I need to do a better job of that guiding people on the easiest path to run a game too.

**Comment by djigneo** [comment]
Reorganized content:
- Modified Character issue template to be a simplified checklist with links to other resources.
- Split out guidelines / rules around character creation from in-depth character-building content
- Created separate *md file for wiki template page

Hopefully this gets us a little closer to organized. Welcome comments on updated organization.

File: .github/ISSUE_TEMPLATE/character.md [hunk 1]
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
...
-_Your character wasn't born yesterday. Where did they come from? Have they raised themselves from the gutters as a child? Were they pampered among the highest royalty? Who the hell is your character?_
-
-_Even though other players can read your character's background, their characters won't act on it unless it is shared with them in game._
-
-## Potential Plot Hooks
-
-_People are flawed. Not even you dear adventurer are perfect! Has your character pissed someone off? Are they outlawed from a certain town for a... misunderstanding? Do they struggle to overcome something: a paranoia, a physical limitation, an addiction?_ 
-
-_This is entirely optional, but depending on the flare the approving GM may award an XP bonus while your character struggles with their demons._
+[ ] Review the [character creation guidelines](character%20creation%20guidelines.md)
```
Comment (crimsonknave) [inline .github/ISSUE_TEMPLATE/character.md:9]:
```suggestion
- [ ] Review the [character creation guidelines](character%20creation%20guidelines.md)
```

File: .github/ISSUE_TEMPLATE/character.md [hunk 2]
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
...
+[ ] [Make a Wiki page](character%20wiki%20page%20outline.md) for your character <fill in link here>
```
Comment (crimsonknave) [inline .github/ISSUE_TEMPLATE/character.md:10]:
```suggestion
- [ ] [Make a Wiki page](character%20wiki%20page%20outline.md) for your character <fill in link here>
```

File: .github/ISSUE_TEMPLATE/character.md [hunk 3]
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
...
+[ ] Make a character sheet on [DndBeyond](www.dndbeyond.com) [fill in link here]
```
Comment (crimsonknave) [inline .github/ISSUE_TEMPLATE/character.md:11]:
```suggestion
- [ ] Make a character sheet on [DndBeyond](www.dndbeyond.com) [fill in link here]
```

File: .github/ISSUE_TEMPLATE/character.md [hunk 4]
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
...
+[ ] Develop a character concept further - See [Step by Step Characters](step%20%20by%20characters.md)
```
Comment (crimsonknave) [inline .github/ISSUE_TEMPLATE/character.md:12]:
```suggestion
- [ ] Develop a character concept further - See [Step by Step Characters](step%20%20by%20characters.md)
```

File: .github/ISSUE_TEMPLATE/character.md [hunk 5]
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
...
+[ ] Create connections and interact with other characters / players in the community within this issue
```
Comment (crimsonknave) [inline .github/ISSUE_TEMPLATE/character.md:13]:
```suggestion
- [ ] Create connections and interact with other characters / players in the community within this issue
```

File: .github/ISSUE_TEMPLATE/character.md [hunk 6]
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
...
+[ ] Join or organize a Session
```
Comment (crimsonknave) [inline .github/ISSUE_TEMPLATE/character.md:14]:
```suggestion
- [ ] Join or organize a Session
```

File: character creation guidelines.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Character creation guidelines
+
+## Character sheets
+Character sheets are hosted on [DnD Beyond](www.dndbeyond.com)
+The character sheets here do not require background or appearance information, as that information should be found on the character's Wiki page. (See [character wiki page outline](character%20wiki%20page%20outline.md).)
+
+## Character level
+You can create a character at any level, but it is at the GMs discretion to run games for the character levels they feel comfortable with.
+General guideline is to create a character between levels 1 and 3.
```
Comment (crimsonknave) [inline character creation guidelines.md:9]: I think this should note that as the party increases in level it is also fine to make a character of a level that others in the campaign are at.

File: step by step characters.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
+# Step-by-step characters
```
Comment (crimsonknave) [inline step by step characters.md:1]: Isn't everything in this file included in the file above already?

File: character creation guidelines.md (hunk 7 above)
Comment (djigneo) [reply character creation guidelines.md]: Done.

File: step by step characters.md (hunk 8 above)
Comment (djigneo) [reply step by step characters.md:1]: No, if you take a look at this branch, _step by step characters_ was split from _character creation guidelines_ into its own file.

Comment (crimsonknave) [reply step by step characters.md:1]: I'm confused. That's the branch this PR is based on, right? The character-creation.md file that's right above still has the content.

**Pull request merged**